### PR TITLE
checklist updates after 001

### PR DESCRIPTION
**Pull request by rkdarst**
- Can you think of anything else to add?

### CODE REVIEW

File: checklist.md [hunk 1]
```diff
@@ -4,9 +4,16 @@
 
 Advance:
 
+1. rkdarst: test microphones
 1. Write session outline
+   - First line of each section is copy-pasteable to youtube TOC,
+     including links.
```
Comment (bast) [inline checklist.md:10]: can we add an example here on how this should ideally look? one entry.

**Comment by bast** [comment]
Looks complete to me. Thanks for doing all this work!

File: checklist.md (hunk 1 above)
Comment (rkdarst) [reply checklist.md]: Updated, is this good?

**Comment by bast** [comment]
Thanks!

**Pull request merged**